project/src/main/webapp/style.css

```diff
@@ -1,4 +1,34 @@
 /* General Styling */
+a {
+  text-decoration: none;
+}
+
+a:hover {
+  color: #f0eded;
+  text-decoration: none;
+}
+
 body {
+  color: #474747;
+  font-family: 'Open Sans', sans-serif;
   text-align: center;
 }
+
+#navigation-bar {
+  background-color: #4788c4;
+  float: left;
+  font-size: 22px;
+  height: 100%;
+  position: fixed;
+}
+
+#navigation-bar a {
```

Comment:
Can you use a different name for the second navigation-bar?

Reply:
This rule styles the links (the `<a>` elements) within the sidebar. Is that what you're referring to?